cli/internal/cmd/auth.go

```diff
@@ -18,7 +19,14 @@ var loginCmd = &cobra.Command{
 	Use:   "login",
 	Short: "Login with an access token",
 	Args:  cobra.NoArgs,
-	Run: func(cmd *cobra.Command, args []string) {
-		auth.Login()
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var err error
+
+		defer func() {
```

Comment:
Maybe we run this in the root command's PersistentPostRun as a generic way to track all commands? Is it difficult to separate flags from flag arguments that way?

Reply:
I earlier added the command there but cobra doesn't guarantee to run the PostRun command on failure of Run command. Therefore, I bundled the track call within the defer func.